src/clj/game/cards/icebreakers.clj

```diff
@@ -470,6 +470,20 @@
                     {:abilities [(break-sub 1 1 "Barrier")
                                  (strength-pump 1 1)]})
 
+   "Cradle"
+   {:abilities [(break-sub 2 0 "Code Gate")]
+    :events {:card-moved {:silent (req true)
+                          :req (req (and (= "Runner" (:side target))
+                                         (= [:hand] (or (:zone target)
+                                                        (:previous-zone target)))))
+                          :effect (effect (update-breaker-strength card))}
+             :runner-draw {:silent (req true)
```

Comment:
Does runner draw not trigger `:card-moved`?

Reply:
It does not.

Reply:
I don't know why, but `draw` doesn't use `move`; it manually updates the state vectors.